src/triage/cli.py

```diff
@@ -213,6 +232,7 @@ def __init__(self, parser):
 
     @cachedproperty
     def runner(self):
+        _ = self.root.setup # Loading configuration (if exists)
```

Comment:
certainly would prefer the statement:

    self.root.setup()

Reply:
Agreed